services/blockchain-indexer/shared/dataService/dpos/delegates.js

```diff
@@ -174,7 +186,17 @@ const getDelegates = async params => {
 		: (acc[entity] && acc[entity] === params[entity]),
 	);
 
-	delegates.data = allDelegates;
+	delegates.data = await BluebirdPromise.map(
+		allDelegates,
+		async delegate => {
+			const [validatorInfo = {}] = await validatorsTable.find({ name: delegate.name });
+			return {
+				...delegate,
+				forgedBlocks: validatorInfo.forgedBlocks || 0,
+				rewards: validatorInfo.rewards || 0,
```

Comment:
Please verify this. I believe this should be a BigInt.
```suggestion
				rewards: validatorInfo.rewards || BigInt('0'),
```